locales/en-US/server.ftl

```diff
@@ -231,6 +231,9 @@ annotationTextConfirmButton =
     .title = Confirm
 annotationTextCancelButton =
     .title = Cancel
+# Texttool input textbox placeholder
```

Comment:
I believe this comment needs rewording, right now it feels like a dev note. Can you point me to a mock-up or screenshot?